packages/calcite-components/src/components/button/button.scss

```diff
@@ -1,4 +1,29 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-button-background-color: Specifies the background color of the component.
+ * @prop --calcite-button-text-color: Specifies the text and icon color of the component.
```

Comment:
We should separate the icon css props - add `--calcite-button-icon-start-color` and `--calcite-button-icon-end-color`, which can default to thi text-color value.

Reply:
I suppose you could also have a `--calcite-button-icon-color` var that would set both, and these explicitly for each, but not sure how far down the rabbit hole we want to go there. Ostensibly, you could set `--calcite-icon-color` on a button to achieve the same as `--calcite-button-icon-color`, so only individual ones for start/end makes sense and doesn't prevent a theming story.

Reply:
Yeah that makes sense 👍